source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -234,44 +277,46 @@ unittest
     auto blocks = node_1.getBlocksFrom(0, 2);
     assert(blocks.length == 1);
 
-    auto sleep_node_1 = nodes[$ - 1];
-    auto sleep_node_2 = nodes[$ - 2];
+    auto sleep_node_1 = nodes[$ - 1]; // node #5
+    auto sleep_node_2 = nodes[$ - 2]; // node #4
```

Comment:
```suggestion
    auto sleep_node_1 = nodes[$ - 2]; // node #4
    auto sleep_node_2 = nodes[$ - 1]; // node #5
```
I think this would be more intuitive. But do we need the comment like "node #5, node #4"?

Reply:
I have added a couple of rename commits to make it clearer.